site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -27,6 +27,7 @@ You will find the executables for the SDK server, one for each type of operating
 - `sdk-server.windows.amd64.exe` - Windows
 - `sdk-server.darwin.amd64` - macOS
 - `sdk-server.linux.amd64` - Linux
+- `docker run --env local=true gcr.io/agones-images/agones-sdk:1.15.0` - Docker (runs in Windows container)
```

Comment:
What do you mean here by "runs in Windows container" ?

Also, if you want to do this, we should also cover how to mount the config file into the container, since the Docker image has no access to the local filesystem otherwise.

Reply:
I just downloaded the zip file and it only contains the 3 files that are already mentioned in the documentation. So as-is, this is a bit misleading. 

It looks like you are suggesting an alternative approach where instead of running the windows executable, you are running the container image in docker configured for local mode.

This sounds like an interesting alternative to what is currently documented but I think it needs more explanation so that developers understand _why_ they would want to do this. When is this preferable to running the sdkserver directly? When is it preferable to using something like minikube or kind if you are running the sdk server and/or game server as a container image?

Reply:
yep, seems should update. use case - you have traditional desktop server - in docker it fails. so when it in docker, need to access agones. windows does not have host docker network, and avoiding patching gameserver is awesome. so can put agones and gameserver into same docker compose (not tried kind/minikube yet). what is awesome - agones sdk has windows docker making that possible, but sure. need update, may be will provide after file cli fix, so can run clean solution e2e https://github.com/googleforgames/agones/pull/2174

Reply:
This can also be run `docker run gcr.io/agones-images/agones-sdk:1.15.0 --local=true` which I think looks cleaner anyway, and means you don't have to wait for the env fix for FILE, you could append `--file` to this command as well.

But I concur with Robbie - this probably warrants it's own "Running with a local container" (or similar naming) section, covering `docker run` and `docker compose`.

If you are going down that path, https://github.com/googleforgames/agones/issues/1183#issuecomment-554820747 might also be a useful trick on how to share networks between the containers.

Reply:
Touching base - just wanted to check in and see if you were planning on updating this PR?